core/domain/summary_services_test.py

```diff
@@ -540,3 +540,109 @@ def test_only_explorations_with_ratings_are_returned(self):
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+
+        self.save_new_valid_exploration(self.EXP_ID_1, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_2, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_3, self.albert_id)
+
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_2)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_1)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_3)
+
+        self.set_admins([self.ADMIN_USERNAME])
+
+    def test_for_recently_published_explorations(self):
+        """ Tests for recently published explorations.
+        """
+
+        recently_published_exploration_summaries = (
+            summary_services.get_recently_published_exploration_summary_dicts([
+                feconf.DEFAULT_LANGUAGE_CODE]))
+        test_summary_1 = {
+            'status': 'public',
+            'thumbnail_bg_color': '#a33f40',
+            'community_owned': False,
+            'tags': [],
+            'thumbnail_icon_url': '/images/subjects/Lightbulb.svg',
+            'language_code': feconf.DEFAULT_LANGUAGE_CODE,
+            'id': self.EXP_ID_1,
+            'category': u'A category',
+            'ratings': feconf.get_empty_ratings(),
+            'title': u'Title 1',
+            'num_views': 0,
+            'objective': u'An objective'
+        }
+        test_summary_2 = {
+            'status': 'public',
+            'thumbnail_bg_color': '#a33f40',
+            'community_owned': False,
+            'tags': [],
+            'thumbnail_icon_url': '/images/subjects/Lightbulb.svg',
+            'language_code': feconf.DEFAULT_LANGUAGE_CODE,
+            'id': self.EXP_ID_2,
+            'category': u'A category',
+            'ratings': feconf.get_empty_ratings(),
+            'title': u'Title 2',
+            'num_views': 0,
+            'objective': u'An objective'
+        }
+        test_summary_3 = {
+            'status': 'public',
+            'thumbnail_bg_color': '#a33f40',
+            'community_owned': False,
+            'tags': [],
+            'thumbnail_icon_url': '/images/subjects/Lightbulb.svg',
+            'language_code': feconf.DEFAULT_LANGUAGE_CODE,
+            'id': self.EXP_ID_3,
+            'category': u'A category',
+            'ratings': feconf.get_empty_ratings(),
+            'title': u'Title 3',
+            'num_views': 0,
+            'objective': u'An objective'
+        }
+
+        self.assertDictContainsSubset(
+            test_summary_3, recently_published_exploration_summaries[0])
+        self.assertDictContainsSubset(
+            test_summary_1, recently_published_exploration_summaries[1])
+        self.assertDictContainsSubset(
+            test_summary_2, recently_published_exploration_summaries[2])
+
+        test_summary_1['title'] = u'New title'
+        self.assertDictContainsSubset(
```

Comment:
It's a good idea, but you actually need to save the exploration and refetch the summaries. You're just modifying the local variable at the moment.